src/components/Button/Button.vue

```diff
@@ -300,6 +309,21 @@ export default {
 	},
 
 	computed: {
+		// Determines whether the root element is an a or a button
+		rootElement() {
+			if (this.href) {
+				return {
+					is: 'a',
+					href: this.href,
+					...this.$attrs,
+				}
+			}
+			return {
+				is: 'button',
+				...this.$attrs,
+			}
```

Comment:
@marcoambrosini I am not 100% sure, but I think `href` should be included in `$attrs`, so this should work as well:
```suggestion
			return {
				is: this.href ? 'a' : 'button',
				...this.$attrs,
			}
```

Reply:
I tested without explicitly adding href and it didn't work 🤷‍♂️

Reply:
maybe cause it's a prop it's captured by $props and taken out of $attrs?